### ISSUE TITLE
Search mishandles last cookie entry

### ISSUE THREAD

**Issue by nicobubulle** [Novik/ruTorrent#626]
_From [david.a...@gmail.com](https://code.google.com/u/114652900312388899229/) on October 07, 2011 00:16:17_

What environment are you using? 1. My ruTorrent version is... 3.3
2. My rTorrent version was installed last week.
3. I use web-server lighttpd on Synology NAS (linux)
4. I use browser Google Chrome it version is 14.0.835.187 m on Win7

My cookies section has three lines. I was trying to search for files on the third site that I had a cookie set for, but it wasn't producing any results. It had worked the previous day. I went into the cookies section and on a programmer's hunch added a carriage return to the end of the cookies (after the semicolon). It fixed the problem.

TL;DR Bug: the cookies section needs an carriage return at the end or the last site will not work.

_Original issue: http://code.google.com/p/rutorrent/issues/detail?id=628_
